jest.config.base.js

```diff
@@ -20,7 +20,11 @@ module.exports = {
     clearMocks: true,
     testMatch: ["**/?(*.)+(spec|test).+(ts|tsx)", "test/**/*.+(ts|tsx)",],
     transform: {
-        "^.+\\.(ts|tsx)$": "@swc/jest"
+        "^.+\\.(ts|tsx)$": "@swc/jest",
+        "^.+\\.(js|jsx)$": "babel-jest"
     },
+    transformIgnorePatterns: [
+        "!node_modules/"
```

Comment:
I don't understand this, you want to `ignore` whatever is `not node_modules`. Which means that you ignore the sources and transpile only what is in `node_modules`? 🙃

Reply:
Fixed